Makefile

```diff
@@ -14,6 +14,7 @@ build:
     --require=asciidoctor-bibtex \
     --require=asciidoctor-diagram \
     --require=asciidoctor-mathematical \
+    --require=$(SAIL_ASCIIDOC_PLUGIN) \
```

Comment:
Where is this defined in ? Do we need to add anything to the CI file to fetch it ?

Reply:
Ah, I had it like that when I was developing the plugin so I could use a local copy of my gem. I now have a RubyGem uploaded [here](https://rubygems.org/gems/asciidoctor-sail), so I can update it to `--require=asciidoctor-sail` (although I may need to push a new version).

Reply:
Yes please do, thank you.

Reply:
We would also need to add that gem to the [dependencies file]( https://github.com/Alasdair/riscv-code-size-reduction/blob/main/dependencies/Gemfile)

I changed that line and added the SAIL gem to dependencies file in my fork and it appears that SAIL plugin is being installed correctly, but JSON parsing is failing for some reason or another ? https://github.com/abukharmeh/riscv-code-size-reduction/actions/runs/4666410899/jobs/8260972501

Are you using a different version of Asciidoctor while testing from the one being used by CI ?